lib/routes/account.js

```diff
@@ -484,13 +498,17 @@ module.exports = function (
                   uid: emailRecord.uid,
                   kA: emailRecord.kA,
                   wrapKb: wrapKb,
-                  emailVerified: emailRecord.emailVerified
+                  emailVerified: emailRecord.emailVerified,
+                  tokenVerificationId: tokenVerificationId,
+                  tokenVerified: false
```

Comment:
It seems odd to send the email with the code in it, before we've created one of the tokens that the code is supposed to verify.  I'm sure nobody would ever hit that race condition in practice, but I wonder if we can refactor this to create the `keyFetchToken` before sending the email?